src/storage/txn/commands/macros.rs

```diff
@@ -11,6 +11,21 @@ macro_rules! ctx {
     };
 }
 
+/// Generate the struct definition and Debug, Display methods for a passed in
+/// storage command. An instance of this type is created from the `From`
```

Comment:
This is not quite correct, Commands are created from grpc messages, the `From` impl is for conversion between typed and untyped versions.

Reply:
I need some clarification here: From what I can make out, both the Command enum variant and the Command struct instances are created in this flow that I've described above (Steps 3 and Steps 4 in my doc). Which is the part of the codebase that creates a Command from the grpc request?

Reply:
I think a Command is only created at the same time as a TypedCommand at the moment, but there is no reason a Command should not be created without a TypedCommand (this is done in tests, for example).

Reply:
Ah, make sense. I've removed that line. I just checked that the usage of the macro can be figured out using IntelliJ's Find Usages also. And now there is enough documentation around this piece, removing that one line won't lose much.